kura/org.eclipse.kura.core.net/META-INF/MANIFEST.MF

```diff
@@ -15,7 +15,7 @@ Import-Package: org.eclipse.kura;version="[1.0,2.0)",
  org.eclipse.kura.net.dhcp;version="[1.0,2.0)",
  org.eclipse.kura.net.firewall;version="[1.0,2.0)",
  org.eclipse.kura.net.modem;version="[2.1,2.2)",
- org.eclipse.kura.net.wifi;version="[2.0,2.1)",
+ org.eclipse.kura.net.wifi;version="[2.0.0,2.2.0)",
```

Comment:
The format that we usually adopt is [2.0,2.1) without the bug fix version.
Since in this case the core.net bundle implements the net.api the range is restricted to a specific version, that is the reason it was [2.0,2.1) meaning that implements version 2.0 of the APIs. If in this case the API version has been pushed to 2.1 than the version should change to [2.1,2.2)

Reply:
OK. I will update it.